src/dsp_tools/commands/excel2json/models/json_header.py

```diff
@@ -123,12 +115,19 @@ def get(self) -> dict[str, Any]:
             "password": self.password,
             "lang": self.lang,
             "status": True,
-        }
+        } | self.role.to_dict()
+        return usr_dict
+
+
+@dataclass
+class UserRole:
+    project_admin: bool = False
+    sys_admin: bool = False
+
+    def to_dict(self) -> dict[str, list[str]]:
         if self.sys_admin:
-            usr_dict["groups"] = ["SystemAdmin"]
-            usr_dict["projects"] = [":admin", ":member"]
-        elif self.project_member:
-            usr_dict["projects"] = [":member"]
+            return {"groups": ["SystemAdmin"], "projects": [":admin", ":member"]}
```

Comment:
I did it this way because you never create a sys admin on prod during a `create` but through the app. So if more than one project is on a test server, then it is nice that the person from dasch can also find the project easily... So i saw no harm